x-pack/test/apm_api_integration/basic/tests/transaction_groups/error_rate.ts

```diff
@@ -5,22 +5,21 @@
  */
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../common/ftr_provider_context';
-import expectedErrorRate from './expectation/error_rate.json';
 
 export default function ApiTest({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
 
   // url parameters
-  const start = encodeURIComponent('2020-06-29T06:45:00.000Z');
-  const end = encodeURIComponent('2020-06-29T06:49:00.000Z');
+  const start = encodeURIComponent('2020-08-25T12:51:30.000Z');
+  const end = encodeURIComponent('2020-08-25T12:54:30.000Z');
```

Comment:
You managed to update the existing archive?

Reply:
yes, I did, I did as we talked, I reindexed 10 minutes of data, added the new field `event.outcome` and use the esarchiver CLI to extract the data. After that I added it to the end of the existing archive.

Reply:
@cauemarcondes is this from the obs clusters? did/can you also include .apm-* and .kibana (only the default) indices?

Reply:
@dgieselaar Yes, I took the snapshot from `dev-next-oblt.elastic.dev`. I only included .apm-8.0.0-transaction-0001 index. Do you want me to do it?

Reply:
Also, I noticed that transaction duration metrics are no longer working. I'm hoping a snapshot update would include transaction duration metrics. Looks like it stopped around midnight: `2020-08-25T01:38:46.975Z`. Can you move the time range of the snapshot back a little so it includes transaction duration metrics?

(another request would be to have the time range somewhere as a constant that we can import, rather than copying/pasting it to all the tests)

Reply:
> Also, I noticed that transaction duration metrics are no longer working

What do you mean it's not working? the tests?

Reply:
The data is not being collected on the obs clusters due to a change in APM Server. It's being fixed now, but the snapshot you've collected doesn't have transaction duration metrics. And yes, I'd prefer to have all APM data for that time range, including the .apm-* and .kibana indices. This would allow us to also write integration tests for telemetry, for instance.

Reply:
@sqren how do you feel about it?

Reply:
> The data is not being collected on the obs clusters due to a change in APM Server. It's being fixed now, but the snapshot you've collected doesn't have transaction duration metrics. And yes, I'd prefer to have all APM data for that time range, including the .apm-* and .kibana indices. This would allow us to also write integration tests for telemetry, for instance.

Ah, I got it, I can take a full snapshot from that time range plus .kibana index. Has APM Server already fixed the problem ? can I take a new snapshot now?

Reply:
@cauemarcondes If you go back 48hrs that should be fine. indices would be `.kibana,.apm-*,apm-*`. thanks 🙏

Reply:
Issue is being tracked here: https://github.com/elastic/observability-test-environments/pull/799#event-3694756178

Reply:
> @sqren how do you feel about it?

Sounds good to me 👍 
We should be mindful of the snapshot size (currently around 90kb) but I don't expect the added indicies (`.apm-*` and `.kibana`) to cause a big size increase.

Reply:
Aye, I did see several snapshots weighing close to 10mb in the repo so we've got some wiggle room.

Reply:
Wow, tragedy of the commons...